orchestration/tests/test_orchestrate_unified_args.py

```
@@ -0,0 +1,495 @@
+        normalized_context = context_content.strip()
+        enhanced_task = f"{task_description}\n\n---\n## Pre-computed Context\n{normalized_context}"
+
+        self.assertIn("Fix the authentication bug", enhanced_task)
+        self.assertIn("## Pre-computed Context", enhanced_task)
+        self.assertIn("## Auth Module", enhanced_task)
+        self.assertIn("src/auth.py", enhanced_task)
+
+    def test_task_not_enhanced_without_context(self):
+        """Test that task description is unchanged without context."""
+        task_description = "Simple task"
+        context_content = None
```

Ruff (0.14.10)

Local variable `context_content` is assigned to but never used

Remove assignment to unused variable `context_content`

(F841)